### ISSUE TITLE
Tomcat 7/8 no longer show cert information

### ISSUE THREAD

**Issue by hazendaz** [psi-probe/psi-probe#1000]
Patches to support org/apache/tomcat/util/net/SSLHostConfig cause failure with tomcat 7/8.  We need to allow for backwards compatibility.

**Comment by hazendaz** [comment]
@diogosantana Oops...fixes cause tomcat 7/8 to fail to startup.  I'm also unable to build locally when running tests.

**Comment by hazendaz** [comment]
@diogosantana I added a fix for tomcat 8.5/9.0 support to work and now container starts with tomcat 7/8. I'm not populating anything yet on tomcat 7/8 for certificate information.  It will need to gather that information the old way to put into the new SSLHostConfigInfo pojo you created.  Can you take care of wiring that in?

**Comment by diogosantana** [comment]
Site.

Em 19/02/2017 16:08, "Jeremy Landis" <notifications@github.com> escreveu:

**Comment by diogosantana** [comment]
I mean sure. It's the brazilian portuguese spell check on my phone.

**Comment by hazendaz** [comment]
Great!  Thanks.  I did make a small patch for suspected memory leaks highlighted by eclipse (not with this but elsewhere).  I had to back off one as it broke startup.  So if you happen to run into anything broken, it might have to do with that.  I did a once over through entire app and everything seems ok.

Also, I had the unit tests disabled by accident on travis ci (copy paste from a windows only build to get coverity setup).  I have restored that and as a result fixed the issues introduced with the new truststore you added so all good there.

So to recap, provided you don't run into any surprises due to me making other changes, we just need to get tomcat 7/8 to populate new ssl host info so it shows on page.
